sympy/integrals/__init__.py

```diff
@@ -9,7 +9,11 @@
     >>> integrate(sin(x),x)
     -cos(x)
 """
+from .deltafunctions import deltaintegrate
+from .heurisch import heurisch, heurisch_wrapper
 from .integrals import integrate, Integral, line_integrate
+from .manualintegrate import manualintegrate, integral_steps
+from .rationaltools import ratint
```

Comment:
Am 09.03.2015 um 13:58 schrieb Sergey B Kirpichev:
> If the api docs is wrong - you can fix one.  Not with adding new exports.

The relevant question is not how to fix what (I know that well enough 
thank you very much), the relevant question is which needs to be fixed: 
API docs or module.
I was assuming the doc is right (it usually is).

> As I can see in the stable version, the docs examples are
> correct, not wrong:
> http://docs.sympy.org/latest/modules/integrals/integrals.html#api-reference

That's not the independent confirmation that you assume it is, it is 
generated from the "API Reference" section that happens to be within 
this diff.